content/press.adoc

```diff
@@ -94,3 +94,6 @@ volunteers so availability and time may be limited.
 * link:https://github.com/batmat[Baptiste Mathus] (_french, english_) - Contributor / Meetup Organizer - `bmathus[at]batmat.net`
 * link:https://github.com/orrc[Christopher Orr] (_english, german_) - Contributor / Speaker - `chris[at]orr.me.uk`
 
+=== CIS press contacts
```

Comment:
is CIS an understood phrase by journalists? You're the only person I've ever seen actually use it